firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -626,4 +691,14 @@ public ConfigContainer getFetchedConfigs() {
       int LOCAL_STORAGE_USED = 2;
     }
   }
+
+  public enum FetchType {
+    BASE,
```

Comment:
The enum value can be stored in an instance field, which is more clear to read and makes it independent of the enum itself, i.e.

```
BASE("Base"),
...

private final int value;

FetchType(value) {
  this.value = value;
}

String getValue() {
  return value;
}
```

Reply:
Updated.